frontend/src/Style/Display.elm

```diff
@@ -0,0 +1,23 @@
+module Style.Display exposing (block, flex)
+
+
+type DisplayTypes
```

Comment:
This type is unnecessary, just define return the class name in the below functions.